src/client/eppo-client.ts

```diff
@@ -68,6 +68,12 @@ export type FlagConfigurationRequestParameters = {
   skipInitialPoll?: boolean;
 };
 
+export interface IFlagExperiment<T> {
```

Comment:
> We may want to call this something different that is more clear what it is doing and how it is used

`ExperiementContainerExperiment`

> Is this templatized so in case future CMS inputs take on different shapes?

Yes, exactly. This is looking forward a bit to subsequent CMS work like webflow.

Reply:
updated with `IContainerExperiment`, but let me know if you have another suggestion